src/coffee/directives/api/models/child/base-poly-child-model.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('uiGmapgoogle-maps.directives.api')
     (Builder, gFactory) ->
       class BasePolyChildModel extends Builder
         @include GmapUtil
-        constructor: (@scope, @attrs, @map, @defaults, @model) ->
+        constructor: (@scope, @attrs, @map, @defaults, @model, gObjectChangeCb) ->
```

Comment:
the gObjectChange is meant to be a generic change from the direction of GoolgleMaps, so it could be called by anything that calls this callback. In this case the only real thing firing back to the parent is arraySync changes (such as edits).